webapp-django/crashstats/signature/jinja2/signature/signature_report.html

```diff
@@ -13,21 +13,28 @@
 {% block content %}
 <div id="mainbody">
     <div class="page-heading">
+        <a href="#" class="display-toggle-filters toggle-filters show">Show Search Filters</a>
```

Comment:
This is an inline block preceding a block element. I think you should put this in a `<p>` tag and put `.display-toggle-filters` on it instead.